source/js/main.js

```diff
@@ -23,13 +23,14 @@ import fellowships from './fellowships';
 const SHOW_MEMBER_NOTICE = false;
 
 // To be populated via XHR...
-let env, networkSiteURL;
+let env, networkSiteURL, pulseApiURL;
 
```

Comment:
Is this going to cause problems when promoting heroku pipelines?

Reply:
Oh or is it pulling this from the environment json view that the server exposes?

Reply:
@alanmoo I think it should be fine. I'm following exactly how we are doing to get `networkSiteURL` from the env var `NETWORK_SITE_URL`.